Assets/MixedRealityToolkit.Examples/Demos/UX/Interactables/Scenes/InteractablesGallery.unity

```diff
@@ -320,7 +320,7 @@ Prefab:
     - target: {fileID: 114718788154663760, guid: 51cc6641d88b49d46bd38572540efe6c,
         type: 2}
       propertyPath: Events.Array.data[0].AssemblyQualifiedName
-      value: Microsoft.MixedReality.Toolkit.SDK.UX.Interactable.Events.InteractableAudioReceiver,
+      value: Microsoft.MixedReality.Toolkit.UI.InteractableAudioReceiver,
         Microsoft.MixedReality.Toolkit.SDK
```

Comment:
Was this the only scene that changed?

Reply:
Appears to be, I did do a full text search over all files under MRTK looking for these to update.

---
In reply to: [267116879](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3658#discussion_r267116879) [](ancestors = 267116879)